package.json

```diff
@@ -48,7 +48,6 @@
   },
   "devDependencies": {
     "cross-env": "^5.2.1",
-    "cypress": "^3.4.1",
```

Comment:
Thanks!  I saw this almost *6 minutes* install time (just the `npm install` portion; see last 2 records):
```json
{
  "type": "event",
  "category": "user",
  "action": "download::getApp",
  "label": "staminal-dove-12",
  "principal": "323795683-root",
  "level": "info",
  "message": "analytics",
  "timestamp": "2019-09-29T07:28:05.633Z"
}
{
  "type": "event",
  "category": "user",
  "action": "download::download",
  "label": "staminal-dove-12",
  "principal": "323795683-root",
  "level": "info",
  "message": "analytics",
  "timestamp": "2019-09-29T07:28:06.213Z"
}
{
  "type": "event",
  "category": "user",
  "action": "download::build",
  "label": "staminal-dove-12",
  "principal": "323795683-root",
  "level": "info",
  "message": "analytics",
  "timestamp": "2019-09-29T07:28:06.214Z"
}
{
  "type": "event",
  "category": "user",
  "action": "download::install",
  "label": "staminal-dove-12",
  "principal": "323795683-root",
  "level": "info",
  "message": "analytics",
  "timestamp": "2019-09-29T07:28:09.430Z"
}
{
  "type": "event",
  "category": "user",
  "action": "download::done",
  "label": "staminal-dove-12",
  "principal": "323795683-root",
  "level": "info",
  "message": "analytics",
  "timestamp": "2019-09-29T07:36:00.357Z"
}
```

Please kill it with fire.